samples/source/README.md

```diff
@@ -2,11 +2,11 @@
 
 ## Prerequisites
 
+Same as listed [here](../../source/DEVELOPMENT.md#prerequisites)
```

Comment:
What are the pre-requisites from the development side required for trying this sample ?